src/bin/rustfmt.rs

```diff
@@ -20,14 +20,16 @@ extern crate getopts;
 use rustfmt::{run, Input};
 use rustfmt::config::{Config, WriteMode};
 
-use std::env;
+use std::{env, error};
 use std::fs::{self, File};
 use std::io::{self, ErrorKind, Read, Write};
 use std::path::{Path, PathBuf};
 use std::str::FromStr;
 
 use getopts::{Matches, Options};
 
+type Error = Box<error::Error + Send + Sync>;
```

Comment:
In which case can we have FmtError rather than just Error for the same reason as Result below.